snowpack/src/build/process.ts

```diff
@@ -111,7 +111,7 @@ export async function createBuildState(commandOptions: CommandOptions): Promise<
   const isHMR = getIsHmrEnabled(config);
 
   // Seems like maybe we shouldn't be doing this...
-  config.buildOptions.resolveProxyImports = !config.optimize?.bundle;
+  config.buildOptions.resolveProxyImports = true;
```

Comment:
@matthewp (See PR description for full explanation): Changing this feels like it’d have some implications, but we need to pass at least `*.module.css.proxy.js` files to esbuild. What was the original intent of this setting? And can it be changed to always true? Or should we find a way for `*.module.css.proxy.js` to just be passed through no matter what?